packages/lodestar-cli/test/unit/options/beaconNodeOptions.test.ts

```diff
@@ -64,6 +65,7 @@ describe("options / beaconNodeOptions", () => {
         pushGateway: true,
         serverPort: 8765,
         timeout: 5000,
+        listenAddr: "0.0.0.0",
```

Comment:
I feel like 127.0.0.1 would be more sensible default value

Reply:
I just used 0.0.0.0 because that's what the node http server uses by default.